public/index.html

```diff
@@ -9,10 +9,30 @@
     />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Make an appointment to get your COVID vaccine"
-    />
+    <meta name="keywords" content="massachusetts, mass, ma, covid, covid-19, vaccine, vaccines, vax, search, sign up, zipcode">
```

Comment:
All of these need a self close, right?   <meta ... />

Reply:
according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta), `<meta>` is a "void element" which means no children and no end tag are allowed.  so `<meta>` with no closing `/>` is correct.

Reply:
I added closing tags for all meta